### PR TITLE
Added wording for PPV2 support

### PR DESCRIPTION
**Pull request by ellioman**
### Purpose of this PR
Updated documentation with information on PPV2 support.

Redoing this PR #5668 as that one was accidentally aimed at Master.

### CODE REVIEW

File: com.unity.render-pipelines.universal/Documentation~/InstallURPIntoAProject.md [hunk 1]
```diff
@@ -4,9 +4,9 @@ You can download and install the latest version of the Universal Render Pipeline
 
 ## Before you begin
 
-URP uses its own [integrated post-processing solution](integration-with-post-processing.md). If you have the Post Processing Version 2 package installed in your Project, you need to delete the Post Processing Version 2 package before you install URP into your Project. When you have installed URP, you can then recreate your post-processing effects.
+URP supports both [Post Processing Version 2 (PPV2) and its own integrated post-processing solution](integration-with-post-processing.md). If you have the Post Processing Version 2 package installed in your Project and you want to use URP's integrated post-processing solution, you need to delete the Post Processing Version 2 package before you install URP into your Project. When you have installed URP, you can then recreate your post-processing effects.
```
Comment (phi-lira) [inline com.unity.render-pipelines.universal/Documentation~/InstallURPIntoAProject.md]: Suggestion, reword this to say URP supports an integrated post-processing stack. PPv2 is deprecated.

File: com.unity.render-pipelines.universal/Documentation~/InstallURPIntoAProject.md [hunk 2]
```diff
@@ -4,9 +4,9 @@ You can download and install the latest version of the Universal Render Pipeline
...
 
-URP does not currently support custom post-processing effects. If your Project uses custom post-processing effects, these cannot currently be recreated in URP. Custom post-processing effects will be supported in a forthcoming release of URP.
+URP's integrated post-processing solution does not currently support custom post-processing effects. If your Project uses custom post-processing effects, these cannot currently be recreated in URP's integrated post-processing solution. Custom post-processing effects will be supported in a forthcoming release of URP.
```
Comment (phi-lira) [inline com.unity.render-pipelines.universal/Documentation~/InstallURPIntoAProject.md]: `custom post-processing effects made for PPv2`.

File: com.unity.render-pipelines.universal/Documentation~/integration-with-post-processing.md [hunk 3]
```diff
@@ -1,7 +1,13 @@
 # Post-processing in the Universal Render Pipeline
 
-The Universal Render Pipeline (URP) includes its own implementation of [post-processing](https://docs.unity3d.com/Manual/PostProcessingOverview.html), so you do not need to install any other package. URP is not compatible with the post-processing version 2 package. 
+The Universal Render Pipeline (URP) supports 2 [post-processing](https://docs.unity3d.com/Manual/PostProcessingOverview.html) solutions: Unity's Post Processing Version 2 (PPV2) package, and URP's own integrated solution.
```
Comment (phi-lira) [inline com.unity.render-pipelines.universal/Documentation~/integration-with-post-processing.md]: mention that PPv2 is supported as fallback/deprecation. integrated post-processing is recommended for best performance and experience.

File: com.unity.render-pipelines.universal/Documentation~/universalrp-builtin-feature-comparison.md [hunk 4]
```diff
@@ -121,7 +121,7 @@ This table provides an overview of the current features supported in the Univers
 | UniversalRenderPipeline.RenderSingleCamera                   | Not supported                                                | Yes                                                          |
 | ScriptableRenderPass                                         | Not supported                                                | Yes                                                          |
 | Custom Renderers                                             | Not supported                                                | Yes                                                          |
-| ***Post-processing***                                        | Uses Post-Processing Version 2 package                      | Uses integrated [post-processing solution](integration-with-post-processing.md) |
+| ***Post-processing***                                        | Uses Post-Processing Version 2 package                      | Uses Post-Processing Version 2 package and integrated [post-processing solution](integration-with-post-processing.md) |
```
Comment (phi-lira) [inline com.unity.render-pipelines.universal/Documentation~/universalrp-builtin-feature-comparison.md]: Uses integrated post processing. PPv2 is deprecated.

**Comment by reallyfancy** [comment]
@phi-lira What do you think of the changes I've made? Is this sufficiently clear?

**Comment by phi-lira** [comment]
LGTM.
